tests/ops/qubit/test_matrix_ops.py

```diff
@@ -983,13 +984,22 @@ def test_blockencode_torch(self, input_matrix, wires, output_matrix):
     )
     def test_blockencode_jax(self, input_matrix, wires, output_matrix):
         """Test that the BlockEncode operator matrix is correct for jax."""
+        import jax
         import jax.numpy as jnp
 
         input_matrix = jnp.array(input_matrix)
         op = qml.BlockEncode(input_matrix, wires)
         assert np.allclose(qml.matrix(op), output_matrix)
         assert qml.math.get_interface(qml.matrix(op)) == "jax"
 
+        # Test jitting behaviour as well.
+        @jax.jit
+        def f(A):
+            op = qml.BlockEncode(A, wires)
+            return qml.matrix(op)
+
+        assert np.allclose(f(input_matrix), output_matrix)
```

Comment:
Does this test that it was jitted correctly? What if it didn't jit anything under the hood and just used the default jax execution? 

Asking for my own curiosity, how does one usually test that things are actually being jitted?

Reply:
Hmm, good question. Maybe we should just trust that if `jax.jit` is applied to the function it does indeed jit 😅